changelog/tt_udon.md

```diff
@@ -0,0 +1,3 @@
+### Ignored
+
+- Update spec tests
```

Comment:
I think this is a breaking change for minimal config and should be highlighted as a change.

Reply:
agree, pushed to fix that